content/articles/2018-10-devtools-2.0.0.Rmarkdown

```diff
@@ -0,0 +1,256 @@
+---
+title: devtools 2.0.0
+date: 2018-10-26
+slug: devtools-2-0-0
+author: Jim Hester
+categories: [package]
+description: >
+    Make package development easier by providing R functions that simplify and expedite common tasks.
+photo:
+  url: https://unsplash.com/photos/wEJK4q_YlNQ
+  author: Hunter Haley
+---
+
+## Introduction
+
+[devtools] 2.0.0 is now on CRAN!
+
+devtools makes package development easier by providing R functions that
+simplify and expedite common tasks. [R Packages] is a book based around this
+workflow.
```

Comment:
Do you think we should address the fact that R Packages is slated for revision and refers to an older version of devtools?